Packs/Troubleshoot/Scripts/TroubleshootExecutePlaybookByAlertQuery/TroubleshootExecutePlaybookByAlertQuery.py

```diff
@@ -18,19 +18,31 @@ def __init__(self, playbooks_dict):
             "others": []
         }
 
-    def update_success(self, playbook_id: str, alert_ids: list):
+    def update_success(self, playbook_id: str, alert_ids: str | list):
         """Update the 'success' dictionary with alert IDs for the given playbook ID."""
-        if playbook_id in self.results_summary["success"]:
-            self.results_summary["success"][playbook_id].extend(alert_ids)
+        if isinstance(alert_ids, str):
```

Comment:
If its a string wrap it with a list and continue with the same code as before, no need for the outer if else.